tvision/Makefile.bcc

```diff
@@ -8,13 +8,14 @@
 # I used BC++ 5.5 free command line tools for testing.
 #
 
+ExLIB=.dll
 !ifdef FOR_NMAKE
+DEFINE_FOR_NMAKE=FOR_NMAKE=$(FOR_NMAKE)
```

Comment:
Microsoft nmake differs from GNU make in the treatment of empty defines.  With GNU make, "FOR_NMAKE=$(FOR_NMAKE)" will result in FOR_NMAKE being undefined if $(FOR_NMAKE) is empty.  However, with MS nmake, if $(FOR_NMAKE) is empty, then "FOR_NMAKE=$(FOR_NMAKE)" will result in FOR_NMAKE being defined but containing no value.  Thus, with nmake, to prevent the parameter from being defined if the value does not exist, we must make sure not to include the parameter at all.  That's what this DEFINE_FOR_NMAKE does.